recipes/libpcsclite/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pcsclite.apdu.fr/files/pcsc-lite-{{ version }}.tar.bz2
+  sha256: cbcc3b34c61f53291cecc0d831423c94d437b188eb2b97b7febc08de1c914e8a
+
+build:
+  skip: True  # [not linux]
+  script:
+    - set -e
+    - set -x
+    - if test -f Makefile; then make distclean; fi
+    - rm -rf *.cache *.m4 config.guess config.log config.status config.sub depcomp ltmain.sh
+    - autoreconf --verbose --install --force
+    - ./configure --prefix=$PREFIX --sbindir=$PREFIX/bin
+    - make install
+    - make distcheck
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('pcsclite') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('systemd-devel') }}
+    - autoconf-archive
+    - automake
+    - file
+    - flex
+    - libtool
+    - make
+    - pkg-config 
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - libudev
+  run:
+
+test:
+  commands:
+    - pcscd --version
+    - test -f $PREFIX/include/PCSC/pcsclite.h
+    # Test specific SO versions to prevent future package ABI breaks
+    - test -f $PREFIX/lib/libpcsclite.so.1
+    - test -f $PREFIX/lib/libpcscspy.so.0
+    - test -f $PREFIX/lib/pkgconfig/libpcsclite.pc
```

Comment:
@carterbox why do we care about this? It has the host prefix hard-coded which is useless.

Reply:
Generally checking for CMake/Package config files indicates that they are being shipped without needing to download the package and look. Downstream builds can break if they depend on config files which are here today and removed later.

What do you mean useless? [Paths like `/home/conda/staged-recipes/build_artifacts/pcsclite_1672093677527/_h_env_placehold_placehold_placehold_` are replaced by conda with `$CONDA_PREFIX` at install-time.](https://docs.conda.io/projects/conda-build/en/latest/resources/make-relocatable.html)

Reply:
You're right, I was being silly about the placeholder prefix. 

Still, I rarely see recipes check for .pc files (plz don't mix/compare them with cmake files; they are of vastly different values) and I can raise many counter examples, so if you think it's the standard practice it'd be better to document it.

Reply:
I'm not commenting on the relative merits of CMake vs Package Config

If upstream developers of this package want to use PC, then we should ship them too. Testing whether they are there indicates to downstream users that we are including all of upstream's artifacts.